portal-impl/src/portal.properties

```diff
@@ -68,7 +68,7 @@
     #
     # Env: LIFERAY_PORTAL_PERIOD_INSTANCE_PERIOD_PROTOCOL
     #
-    portal.instance.protocol=
+
```

Comment:
@jeffwu0724 Please also remove the comments above and below. They are related to the removed property.